src/EditorFeatures/CSharp/StringCopyPaste/StringCopyPasteData.cs

```diff
@@ -42,6 +42,8 @@ internal sealed class StringCopyPasteData(ImmutableArray<StringCopyPasteContent>
         if (string.IsNullOrWhiteSpace(json))
             return null;
 
+        Contract.ThrowIfNull(json);
```

Comment:
looks like we lost NRT annotations on IsNullOrWhitespace.

Reply:
This might be a side effect of us dropping the net8.0 target. If you're able to add that back in that'd be good, but I imagine there's problems there.

Reply:
i'm a little confused how to add that.  doesn't this need to be net472?